### PR TITLE
feat(transport authority): new applications (base)

### PR DESCRIPTION
**Pull request by johannaagma**
# ...

Creating basic applications for each new application for Samgöngustofa (total of 9 applications)

## What

Add necessary files in libs/application/templates and libs/application/template-api/modules for these applications

## Why

Might be a while untill all applications are ready, and we will be working on some applications simultaneously (undecided which applications that will be). Thought it would be better to get the base up for each application, instead of creating a humongous PR after a few months.
Also helps if different people need to work on different applications at the same time

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by saevarma — COMMENTED** [review]
Core files looks okay.
But the application templates should use a feature flag instead of the deprecated `readyForProduction` property.

File: libs/application/templates/reference-template/README.md [hunk 1]
```diff
@@ -6,11 +6,12 @@ This library is a reference how all application template libraries can be.
 
 There are multiple requirements needed for a new template to be usable by the application system:
 
-1. Add a unique application type to `application/core/src/types/ApplicationTypes.ts`
-2. Run `yarn generate @nrwl/react:library application/templates/NAME_OF_APPLICATION` to generate a new library.
-3. The default export of this library has to be an object that extends the `ApplicationTemplate` interface
+1. Run `yarn generate @nrwl/react:library application/templates/NAME_OF_APPLICATION` to generate a new library.
+2. The default export of this library has to be an object that extends the `ApplicationTemplate` interface
+3. Add a unique application type to `application/types/src/lib/ApplicationTypes.ts`
```
Comment (Thorrik58) [inline libs/application/templates/reference-template/README.md:11]: Nice one!

File: libs/application/templates/transport-authority/anonymity-in-vehicle-registry/src/lib/AnonymityInVehicleRegistryTemplate.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        meta: {
+          name: 'Nafnleynd í ökutækjaskrá',
+          actionCard: {
+            tag: {
+              label: m.actionCardDraft,
+              variant: 'blue',
+            },
+          },
+          progress: 0.25,
+          lifecycle: {
```
Comment (Thorrik58) [inline libs/application/templates/transport-authority/anonymity-in-vehicle-registry/src/lib/AnonymityInVehicleRegistryTemplate.ts:46]: Can you use the EphermalStateLifeCycle instead from the application core?

File: libs/application/templates/transport-authority/change-co-owner-of-vehicle/src/lib/ChangeCoOwnerOfVehicleTemplate.tsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+          name: 'Breyting meðeiganda á ökutæki',
+          actionCard: {
+            tag: {
+              label: m.actionCardDraft,
+              variant: 'blue',
+            },
+          },
+          progress: 0.25,
+          lifecycle: {
+            shouldBeListed: false,
```
Comment (Thorrik58) [inline libs/application/templates/transport-authority/change-co-owner-of-vehicle/src/lib/ChangeCoOwnerOfVehicleTemplate.tsx:47]: same as above

File: libs/application/templates/transport-authority/change-operator-of-vehicle/src/lib/ChangeOperatorOfVehicleTemplate.ts [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+          name: 'Breyting umráðamanns á ökutæki',
+          actionCard: {
+            tag: {
+              label: m.actionCardDraft,
+              variant: 'blue',
+            },
+          },
+          progress: 0.25,
+          lifecycle: {
+            shouldBeListed: false,
```
Comment (Thorrik58) [inline libs/application/templates/transport-authority/change-operator-of-vehicle/src/lib/ChangeOperatorOfVehicleTemplate.ts:47]: same as above

File: libs/application/templates/transport-authority/digital-tachograph-company-card/src/lib/DigitalTachographCompanyCardTemplate.ts [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+          name: 'Fyrirtækjakort',
+          actionCard: {
+            tag: {
+              label: m.actionCardDraft,
+              variant: 'blue',
+            },
+          },
+          progress: 0.25,
+          lifecycle: {
+            shouldBeListed: false,
```
Comment (Thorrik58) [inline libs/application/templates/transport-authority/digital-tachograph-company-card/src/lib/DigitalTachographCompanyCardTemplate.ts:47]: same as above

**Review by saevarma — APPROVED** [review]
Core files are ✅

**Review by Thorrik58 — APPROVED** [review]

**Pull request merged**